### ISSUE TITLE
browse_website using Selenium Chromium opens the url `data:,` then gets stuck

### ISSUE THREAD

**Issue by PeterSR** [Significant-Gravitas/Auto-GPT#1488]
### Duplicates

- [X] I have searched the existing issues

### Steps to reproduce 🕹

My environment:
- OS: Ubuntu 
- Python: 3.8
- Browsers installed: Firefox, Chromium
- Memory type: PineconeMemory
- Not using GPT-4

Run with prompt / ai settings (see below).

- AI first performs google search which runs fine. Then it tries to browse a website:
- Next action: `NEXT ACTION:  COMMAND = browse_website ARGUMENTS = {'url': 'https://www.amazon.com/Keychron-Wireless-Mechanical-Hot-Swappable-Programmable/dp/B0BHZ524M8', 'question': 'reviews'}`
- Chromium window opens (via Selenium) and the url is `data:,`
- Then nothing happens.

Tested on both stable (60b2b61b52c263dba25a6c33623561273e890229, v0.2.0) and master (6a93537c426759708f0e91a125587512c05f354c)

### Current behavior 😯

- Chromium window opens (via Selenium) and the url is `data:,`
- Then nothing happens.

### Expected behavior 🤔

- Chromium window opens with the intended url 
- Some kind of browsing or extraction of the page content
- Further actions

### Your prompt 📝

```yaml
ai_goals:
- Search the web and find good options for a QMK / VIA mechanical keyboard PCB that
  has a wireless bluetooth module. The form factor should be 60%, 75% or 80%
ai_name: A helpful assistant
ai_role: an AI designed to help me solve various tasks
```

**Comment by PeterSR** [comment]
Running in debug mode gives the following extra information:

```
  System: Command browse_website returned: Error: Message: unknown error: DevToolsActivePort file doesn't exist
Stacktrace:
#0 0x55bca6875fe3 <unknown>
#1 0x55bca65b4d36 <unknown>
#2 0x55bca65e1459 <unknown>
```

I know that I have recently been using the dev tools in that browser, but I am not sure how that could affect anything.

**Comment by PeterSR** [comment]
It seems to be a Selenium or ChromeDriver issue. Perhaps an issue with Chromium version or something. But the question is: Can we allow users to change the driver to Firefox / Gecko? Or use a fallback approach?

**Comment by sixtay** [comment]
Can confirm I am getting the same issue.

**Comment by mustfit09** [comment]
I also have the same problem and the environment is the same as him:

- OS: Ubuntu
- Python: 3.8
- Browsers installed: Chromium

**Comment by Pwuts** [comment]
Closing as resolved in #1473
